src/completions/Tab.ts

```diff
@@ -76,8 +74,9 @@ export class BufferCompletionSource extends Completions.CompletionSourceFuse {
             "Tabs",
         )
         this.sortScoredOptions = true
-        this.shouldSetStateFromScore =
-            config.get("completions", "Tab", "autoselect") === "true"
+        this.shouldSetStateFromScore = false
+        // autoselect after space sucks for (fuzzy-) searching with multiple words
+        // config.get("completions", "Tab", "autoselect") === "true"
```

Comment:
As I said on Matrix we shouldn't do this as it will break muscle memory and annoy lots of users. The purpose of this change is to allow users to insert spaces into search queries.

I see three options:

- do nothing, or
- change the `<Space>` bind to `<S-Space>` and document it prominently on the new tab page
- make the `<Space>` bind configurable so it only works on completions where it makes sense (e.g. `:open` and not `:undo` nor `:tab`).